Makefile

```diff
@@ -150,6 +150,11 @@ thrift_gen:
 	$(BUILD)/thrift-gen --generateThrift --inputFile hyperbahn/hyperbahn.thrift --outputDir hyperbahn/gen-go
 	rm -rf trace/thrift/gen-go/tcollector && $(BUILD)/thrift-gen --generateThrift --inputFile trace/tcollector.thrift --outputDir trace/thrift/gen-go/
 
+test_mocks:
+	GOPATH=$(OLDGOPATH) mockery -name TChanSimpleService -recursive -dir ./thrift/gen-go/test/ -output ./thrift/mocks
```

Comment:
Do you mean just as a comment? This needs to be executed before that code is compiled right?